ENTRY_TEMPLATE.yaml

```diff
@@ -78,3 +78,6 @@ licenseUrl: "https://project.org/license/"
 naturalLanguages:
   - English
   - German
+# If there are too many languages to be listed, add '+N languages' above, and a
+# link to the list of languages below
+naturalLanguagesLink: "https://www.transifex.com/org/project/"
```

Comment:
OpenFarm is translated in 62 languages :rofl:

Reply:
Oh wow…

For the website a list would probably still be useful for filters and similar 🤔
And for the README, we don't currently use the languages, right?

Reply:
yeah, we still need to work on #23, but it's still a low priority until we have more projects :)